src/Solver/main.c

```diff
@@ -1,4 +1,25 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include <err.h>
+#include <string.h>
+#include "solver.c"
+#include "grid.c"
```

Comment:
```suggestion
#include "solver.h"
#include "grid.h"
```
Il faut jamais inclure les `.c` mais toujours les `.h` :eyes:

Reply:
alors ça ça s'appelle se précipiter parce que je le sais pertinemment 😅